pyproject.toml

```diff
@@ -26,7 +26,7 @@ packages = [{ include = "pineko", from = "src" }]
 [tool.poetry.dependencies]
 python = ">=3.9,<3.13"
 eko = "^0.14.2"
-pineappl = ">0.7.0"
+pineappl = "^0.8.2"
```

Comment:
I am fine with this. Pineappl is indeed constantly changing with new versions coming with breaking changes. The soon [v0.9.0](https://github.com/NNPDF/pineappl/blob/master/CHANGELOG.md#unreleased) for example will remove `PyOperatorSliceInfo` and `PyPidBasis` in favor of `OperatorSliceInfo` and `PidBasis`, crashing everything in here..

Reply:
> I am fine with this. 

Fine with the commit to keeping track of the changes in pineappl and updating the code here accordingly as it changes or fine with updating the `pyproject.toml` and wait for someone to complain to change it?

(both are fine by me, we are a finite amount of people so at some point some things need to be done in a need-to-use basis)

Reply:
I am also fine with both, but would very much prefer the first option. So in this sense, I have a little preference to sicking to `"^_"`. We can always manually bump the major version of pineappl in a PR and let the CI take care of the crashing (instead of people complaining xd).

Reply:
Ok, that's a good idea. The best of both world. I'll try to prepare such a bot (in the meantime though, plz remember to update pineko when moving to 0.9 :__)